src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -1000,6 +1002,18 @@ private CaseDTO createNewCachedCaseResponse(CachedCase newCase) throws CTPExcept
     return response;
   }
 
+  private void checkCaseTypeForNewAddress(
+      CaseType caseType, uk.gov.ons.ctp.integration.contactcentresvc.representation.Region region)
+      throws CTPException {
+    if ((caseType.name().equals("CE")) && (region.name().equals("N"))) {
```

Comment:
The params to this method are enums. Enums are a type of constant, that also serves to constrain the value assigned to a field of that type.

Enums are there so that we don't litter the code with string literals such as "CE". As a rule of thumb avoid string literals wherever possible.

You can compare an instance of an enum with a reference to an enum value ie

if (caseType == CE)

Reply:
If I put this:

if (caseType == CE)

it gives me this message:

"Incompatible operand types CaseType and String"

Reply:
sorted now - thanks Phil